src/modules/Popup/Popup.js

```diff
@@ -0,0 +1,196 @@
+import React, { Component, PropTypes } from 'react'
+import { findDOMNode } from 'react-dom'
+import { META } from '../../lib'
+import Portal from '../../addons/Portal/Portal'
+import cx from 'classnames'
+import isString from 'lodash/isString'
+import PopupContent from './PopupContent'
+import PopupHeader from './PopupHeader'
+import includes from 'lodash/includes'
+
+const _meta = {
+  name: 'Popup',
+  type: META.TYPES.MODULE,
+}
+
+/**
+ * A <Popup /> displays additional information on top of a page.
+ */
+export default class Popup extends Component {
+
+
+  static propTypes = {
+    /** Display the popup without the pointing arrow */
+    basic: PropTypes.bool,
+
+    /** You may pass a content as children of the Popup */
+    children: PropTypes.node,
+
+    /** Classes to add to the Accordion className. */
+    className: PropTypes.string,
+
+    /** Display the popup on mouse click on the trigger */
+    click: PropTypes.bool,
+
+    /** Simple text content for the popover */
+    content: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+
+    /** A Flowing popup have no maximum width and continue to flow to fit its content */
+    flowing: PropTypes.bool,
+
+    /** Takes up the entire width of its offset container */
+    fluid: PropTypes.bool,
+
+    /** Display the popup on trigger focus */
+    focus: PropTypes.bool,
+
+    /** Header displayed above the content in bold */
+    header: PropTypes.string,
+
+    /** Display the popup on mouse over the trigger */
+    hover: PropTypes.bool,
```

Comment:
When this prop is pulled into the `on` enum prop, could we add `hoverable` here?

![image](https://cloud.githubusercontent.com/assets/5067638/19012148/802a69ca-8761-11e6-827e-a8b76ec13c08.png)